silverpeas/content.md

```diff
@@ -0,0 +1,179 @@
+
+	$ docker run --name silverpeas -p 8080:8000 -d \
+	    --add-host=database:192.168.1.14 \
+	    -v /etc/silverpeas/config.properties:/opt/silverpeas/configuration/config.properties \
+	    -v silverpeas-log:/opt/silverpeas/log \
+	    -v silverpeas-data:/opt/silverpeas/data \
+	    silverpeas
+
+where `database` is the hostname referred by the `DB_SERVER` parameter in your `/etc/silverpeas/config.properties` file as the host running the database system and that is mapped here to the actual IP address of this host. The hostname is added in the `/etc/hosts` file in the container.
+
+For a PostgreSQL database system, some configurations are required in order to be accessed from the Silverpeas container:
+
+-	In the file `postgresql.conf`, edit the parameter `listen_addresses` to add the address of the PostgreSQL host (`192.168.1.14` in our example)
+
+	listen_addresses = 'localhost,192.168.1.14'
+
+-	In the file `pg_hba.conf`, add an entry for the Docker subnetwork
+
+	host all all 172.17.0.0/16 md5
+
+-	Don't forget to restart PostgreSQL for the changes to be taken into account.
+
+# Using a Data Volume Container
+
+To manage more effectively the data produced by an application, the Docker team recommends to use a Data Volume Container. In Silverpeas, there are four types of data produced by the application:
+
+-	the logging stored in `/opt/silverpeas/log`,
+-	the user data and those produced by Silverpeas from the user data in `/opt/silverpeas/data`,
+-	the user domains and the domain authentication definitions in respectively `/opt/silverpeas/properties/org/silverpeas/domains` and `/opt/silverpeas/properties/org/silverpeas/authentication`,
+-	the workflows created by the workflow editor in `/opt/silverpeas/xmlcomponents/workflows`.
+
+Beside these directories, according to your specific needs, custom configuration scripts can be added in the directories `/opt/silverpeas/configuration/jboss` and `/opt/silverpeas/configuration/silverpeas`.
+
+The directories `/opt/silverpeas/log`, `/opt/silverpeas/data`, `/opt/silverpeas/properties`, `/opt/silverpeas/xmlcomponents/workflows`, and `/opt/silverpeas/configuration` are all defined as volumes in the Docker image.
+
+To define a Data Volume Container for Silverpeas, for example:
+
+	$ docker create --name silverpeas-store \
+	    -v silverpeas-data:/opt/silverpeas/data \
+	    -v silverpeas-log:/opt/silverpeas/log \
+	    -v silverpeas-properties:/opt/silverpeas/properties \
+	    -v /etc/silverpeas/config.properties:/opt/silverpeas/configuration/properties \
+	    silverpeas \
+	    /bin/true
+
+Then to mount the volumes in the Silverpeas container:
+
+	$ docker run --name silverpeas -p 8080:8000 -d \
+	    --link postgresql:database \
+	    --volumes-from silverpeas-store \
+	    silverpeas
```

Comment:
What's the value of using a data container over just using an explicit `docker volume create`?